examples/tools/makefile/makedeps/build_project_makefile.rst

```diff
@@ -0,0 +1,156 @@
+
+We'll use `Make <https://www.gnu.org/software/make/>`_ as build system, so you should get it installed
+on Linux and Mac before going forward with this example.
+
+Please, first, clone the sources to recreate this project, you can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ on GitHub:
+
+.. code-block:: shell
+
+    git clone https://github.com/conan-io/examples2.git
+    cd examples2/examples/tools/makefile/makedeps/logger
+
+We start with a very simple C++ language project with the following structure:
+
+.. code-block:: text
+
+    .
+    ├── Makefile
+    ├── conanfile.txt
+    └── src
+        └── main.cpp
+
+This project contains a basic `Makefile <https://www.gnu.org/software/make/manual/make.html#Introduction>`_ including the **spdlog** variable dependency and the
+source code for the logger program in *main.cpp*.
+
+Let's have a look at the *main.cpp* file, it only prints a quote said by Corin, but uses ``spdlog::info`` method for it.
+
+.. code-block:: cpp
+    :caption: **main.cpp**
+
+    #include <cstdlib>
+    #include "spdlog/spdlog.h"
+
+
+    int main(void) {
+        spdlog::info("To be a Cimmerian warrior, you must have both cunning and balance as well as speed and strength.");
+        return EXIT_SUCCESS;
+    }
+
+The *Makefile* specifies that ``logger`` is the expected executable and that it should be linked to the ``spdlog`` and ``fmt`` libraries.
+To simplify the usage, we use global variables to store the paths, flags and libraries.
+
+.. code-block:: make
+    :caption: **Makefile**
+
+    ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+    SRC_FOLDER    = $(ROOT_DIR)/src
+    BUILD_FOLDER  = $(ROOT_DIR)/build
+
+    include $(BUILD_FOLDER)/conandeps.mk
+
+    CXXFLAGS      += $(CONAN_CXXFLAGS) -std=c++11
+    CPPFLAGS      += $(addprefix -I, $(CONAN_INCLUDE_DIRS))
+    CPPFLAGS      += $(addprefix -D, $(CONAN_DEFINES))
+    LDFLAGS       += $(addprefix -L, $(CONAN_LIB_DIRS))
+    LDLIBS        += $(addprefix -l, $(CONAN_LIBS))
+    LDLIBS        += $(addprefix -l, $(CONAN_SYSTEM_LIBS))
+    EXELINKFLAGS  += $(CONAN_EXELINKFLAGS)
+
+
+    all:
+        $(CXX) $(SRC_FOLDER)/main.cpp $(CPPFLAGS) $(CXXFLAGS) $(LDFLAGS) $(LDLIBS) $(EXELINKFLAGS) -o $(BUILD_FOLDER)/logger
+
+The *conanfile.txt* looks simple as it just installs the **spdlog** package and uses **MakeDeps** to build our project.
+
+.. code-block:: ini
+    :caption: **conanfile.txt**
+
+    [requires]
+    spdlog/1.12.0
+
+    [generators]
+    MakeDeps
+
+In this case, we will use :ref:`MakeDeps<conan_tools_gnu_makedeps>` to generate information about where the **spdlog** library
+files are installed thanks to the `makedeps.mk` file and using a `conanbuild[.sh|.bat]` file that describes the compilation environment.
+
+We will use Conan to install **spdlog** library, generate a dependency file for Makefile, for find **spdlog** and **fmt** by make.
```

Comment:
fmt?

Reply:
Removed all references about fmt as it's not out main dependency.